src/components/ui/MasqAlert.jsx

```diff
@@ -0,0 +1,41 @@
+/* global _ */
+import React, { useState } from 'react';
+import nconf from '@qwant/nconf-getter';
+import Alert from 'src/components/ui/Alert';
+
+const masqAlertEnabled = nconf.get().masq.alertEnabled;
+const masqLink = nconf.get().masq.link;
+const masqAlertDate = (new Date(nconf.get().masq.alertDate)).toDateString();
```

Comment:
`toDateString()` does not use the current language.  
A more user-readable format could be used with `Intl.DateTimeFormat`.